sources/sources.bib

```diff
@@ -25,3 +25,17 @@ @article{sample_ref
   publisher={Springer}
 }
 
+@inproceedings{viola_jones,
+  author = {Viola, Paul and Jones, Michael},
+  title = {Remember to Smile: Design of a Mobile Affective Technology to Help Promote Individual Happiness through Smiling},
+  year = {2017},
+}
+
+@inproceedings{opencv_cascade_classifier,
+  author = {OpenCV team},
+  title = {Cascade Classifier},
+  year = {2020},
+  url = {https://docs.opencv.org/3.4/db/d28/tutorial_cascade_classifier.html},
+  note = {Retrieved September 05, 2020}, 
```

Comment:
I listed September 05, 2020 as an example date - if that's when you looked at the website, then disregard this comment. Otherwise, update September 05, 2020 to the date where you looked at the website (presumably September 16, 2020 or something like that)

Reply:
you're absolutely right :D